src/api-service/__app__/agent_commands/__init__.py

```diff
@@ -42,7 +43,13 @@ def delete(req: func.HttpRequest) -> func.HttpResponse:
     return ok(BoolResult(result=True))
 
 
-def main(req: func.HttpRequest) -> func.HttpResponse:
+def main(req: func.HttpRequest, dashboard: func.Out[str]) -> func.HttpResponse:
     methods = {"DELETE": delete, "GET": get}
     method = methods[req.method]
-    return call_if_agent(req, method)
+    result = call_if_agent(req, method)
+
+    events = get_events()
```

Comment:
can we move this logic inside call_if_agent and call_if_user

Reply:
There are a handful of things that I'd like to wrap up into a  streamlined endpoint method.

I'd like to have a method wrapper that handles the req.method->python func mapping, call_if_*, the dashboard, the parsing of the model, and the rendering of a mode.

However, my thought is to do that as a follow-on development effort.

Reply:
Having a standardized wrapper would make it easier to build the swagger specs for the API we've discussed.